### DISCUSSION TITLE
post/disadvantages-of-rss

### DISCUSSION THREAD

**Discussion by giscus** [zihan-ch/blog#26] [General]
# post/disadvantages-of-rss

The introduction of RSS and what the problems of RSS now.

https://zihan.ga/post/disadvantages-of-rss

**Comment by amadeuspagel** [comment]
Can't you use tracking pixels with RSS?

**Reply by arstnei0** [reply]
It doesn't really change the way that RSS works. Even though there are ways to track it, but it's still hard to track the data more granularly.

**Comment by Toniozzzz** [comment]
I use RSS Readers because i'm fed up of ads, pop-ups, ... and tracking (and RSS is such better than Twitter) ! There may be a problem with RSS Readers for interactions but there is a blog's problem too. Most blogs are unreadable, other sites don't have a RSS feed. And when a big firm like Google killed almost RSS twelve years ago, it has not helped to improve RSS technology.
We can discuss how to evolve RSS but it's just a part of a bigger problem : Web 2.0, 3.0, ...
PS : I read this article with my RSS Reader, copy/paste on Firefox to react
